### PR TITLE
Option to turn ticket sales off/on and waiting list on/off

### PR DESCRIPTION
**Pull request by adrianmoisey**
This doesn't turn off the quicket iframe

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -18,3 +18,4 @@ navigation:
 - text: Contact
   url: /contact
 year: 2016
+tickets_for_sale: false
```
Comment (imsickofmaps) [inline _config.yml]: default to `true` for now so we can land it?

Comment (adrianmoisey) [inline _config.yml]: I was thinking it could land when needed?
up to you

File: _config.yml [hunk 2]
```diff
@@ -18,3 +18,6 @@ navigation:
...
+waiting_list_enable: true
+waiting_list_url: https://docs.google.com/forms/d/1WOB3UD0XMfljzwqoqvqIw36xnuDbPs0RcjZDG5_L_4o/viewform?embedded=true
```
Comment (imsickofmaps) [inline _config.yml]: now we're updating this one, can you just add `quicket: 13990` too?

**Comment by adrianmoisey** [comment]
The tickets page looks a little nasty with both iframes in it:
image

**Comment by adrianmoisey** [comment]
Made the quicket stuff configurable, but its getting messy now

File: _layouts/responsive.html [hunk 3]
```diff
@@ -19,18 +19,12 @@
...
-            padding: 10px;
-            border-top: solid 2px #aaa;
-            border-bottom: solid 2px #aaa;
-            margin-top: 15px;
-            color: #222;">
-                <span style="color:#000;">Tickets now open! &nbsp; <a href="/purchase/" class="btn btn-primary">Buy Tickets</a></span>
+   {% if site.tickets_for_sale or site.waiting_list_enable %}
+    <div class="banner" style="text-align: center; background: #fff; font-size: 19px; line-height: 28px; padding: 10px; border-top: solid 2px #aaa; border-bottom: solid 2px #aaa; margin-top: 15px; color: #222;">
+        {% if site.tickets_for_sale %}<span style="color:#000;">Tickets now open! &nbsp; <a href="/purchase/" class="btn btn-primary">Buy Tickets</a></span>{% endif %}
+        {% if site.waiting_list_enable %}<span style="color:#000;">Tickets are sold out! &nbsp; <a href="{{ site.waiting_list_url }}" class="btn btn-primary">Signup for the waiting list</a></span>{% endif %}
```
Comment (vhata) [inline _layouts/responsive.html]: Seems like these two variables are diametric and could be a single boolean.  Or, what you more likely want:

```
if tickets_for_sale:
    if not waiting_list_enable
        Tickets now open!
    else
        Tickets are sold out!
    endif
endif

**Comment by adrianmoisey** [comment]
I think this is tidier/neater.

**Comment by vhata** [comment]
Supermuch tidier :smiley: 

This is really good.

I haven't tested this in jekyll yet but the logic looks sound.

**Pull request merged**